.github/workflows/postsubmit.yml

```diff
@@ -0,0 +1,30 @@
+name: sync-clustersetimage-post
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch: {}
+
+jobs:
+  sync-clusterimageset-release-2_2:
+    name: sync-clusterimageset-release-2.2
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: sync-clusterimageset
+        run: make sync-images-job
+        env:
+          TARGET_BRANCH: release-2.2
+  sync-clusterimageset-release-2_3:
+    name: sync-clusterimageset-release-2.3
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+      - name: sync-clusterimageset
+        run: make sync-images-job
```

Comment:
will this work? I do not think you can directly commit to a release branch with this. Why not use github page publish action?